### ISSUE TITLE
Cool App! I wasn't able to play though....

### ISSUE THREAD

**Issue by donovanrichardson** [wjclavell/Pokemon-Battle#1]
Hi William. I am using Firefox, and the app didn't work. Here's a snippet of what went wrong in my console.
```
NotAllowedError: The play method is not allowed by the user agent or the platform in the current context, possibly because the user denied permission.
ReferenceError: $ is not defined
app.js:39:5
```

I was not able to play in Chrome either, and the only thing that happened in Safari was the music (very nostalgia, btw). Did you use codepen or repl to run this? I look forward to playing this!

**Comment by wjclavell** [comment]
Hey Donovan,

Hmm, that's weird I'll look into it. I coded it in VSCode, and was able to
run it in Chrome just fine.

Thanks for letting me know!

**Comment by donovanrichardson** [comment]
@wjclavell i found the issue, and now I'm able to play! 
<img width="1279" alt="image" src="https://user-images.githubusercontent.com/50427582/85969780-9ee0ff80-b996-11ea-84cd-e4faffcce192.png">
The issue was something to do with the jquery. I had to change the src in pokemon-battle.html
<img width="1159" alt="image" src="https://user-images.githubusercontent.com/50427582/85969295-380f1680-b995-11ea-867e-85b36165f58f.png">
Maybe it's a mac/windows thing. I'm using mac.

**Comment by donovanrichardson** [comment]
@wjclavell When you open the webpage, is the beginning of the url like `http://` or like `file://`? According to [this](https://stackoverflow.com/a/7818464/9608521) answer at Stack Overflow, the script link won't work if the beginning is `file://` like mine is. Maybe I'm starting the webpage wrong? My url is `file://<path to repository>/Pokemon-Battle/pokemon-battle.html`

**Comment by wjclavell** [comment]
Yeah I added that jQuery src yesterday night too! But you're right, it does say file in my URL, anyway to fix that?

**Comment by donovanrichardson** [comment]
@wjclavell if you're opening the web page from your own computer, then I think the url should be ` file://<path to repository>/Pokemon-Battle/pokemon-battle.html`. But if you're hosting it on github, then the url would begin with `https://`. In the case that the url does begin with `https://`, I think the old code would be fine, per the stackoverflow answer I linked above (https://stackoverflow.com/a/7818464/9608521).

**Comment by donovanrichardson** [comment]
Basically, starting the script url with // will cause the webpage to assume that you meant either `file://` or `https://` (whichever begins your webpage's url). But `file://<jquery script>` doesn't exist, hence the problem.
